PLMS/Models/ModelsCategories/GetCategoryViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace PLMS.API.Models.ModelsCategories
+{
+    public class GetCategoryViewModel: CategoryBaseModel
+    {
+        public int Id { get; set; }
+        public string UserId { get; set; } = null!;
+
+        public List<string> GoalTitles { get; set; } = new List<string>();
```

Comment:
Same as in GetCategoryDto.cs - replace string with object

Reply:
fixed